javascript-bank/account.js

```diff
@@ -0,0 +1,43 @@
+/* exported Account */
+
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  var positive = false;
+  var depositObject = new Transaction();
```

Comment:
You shouldn't instantiate this object without the arguments it's intended to take. You should make this object if the amount is an acceptable value, and pass "deposit" and the value as arguments.